src/components/lib/histogram.js

```diff
@@ -86,24 +87,61 @@ export default class Histogram extends Component {
     cutOffRatio: 0, // By default cut off nothing.
   };
 
-  render() {
-    let { top, right, bottom, left, data, width, height, cutOffRatio } = this.props;
+  state = {
+    xScale: (e) => e,
+    yScale: (e) => e,
+    histogramData: []
+  }
+
+  componentWillReceiveProps(nextProps){
```

Comment:
Missing spaces before your `{`s here and below. You can do it with a find replace I think, though don't remember the exact regex off hand.